dmriprep/config/reports-spec.yml

```diff
@@ -91,6 +91,13 @@ sections:
       the final images are resampled using cubic B-Spline interpolation.
     static: false
     subtitle: Alignment of diffusion and anatomical MRI data (surface driven)
+  - bids: {datatype: figures, desc: eddy, suffix: dwi}
```

Comment:
This section should be moved before the alignment with the anatomical data.

Reply:
agree

Reply:
Thanks, fixed in caba8c33655177bc1731edb88ab49a82ad17e213